tox.ini

```diff
@@ -25,6 +25,10 @@ commands =
 	coverage combine
 	coverage report
 
+[flake8]
+max-line-length = 99
+exclude = .tox,.cache,.pytest_cache
```

Comment:
Can you explain why is this needed? Flake is not run in the tox environments, right?

Reply:
Yep you are right in the fact that we do not run flake8 in the tox tests.
However, this was introduced because the pre-commit hook failed (a line was longer than 77 characters), so I had to add global flake8 configurations (placing them in tox.ini works)

Reply:
Interesting, thanks. I think that adding `--max-line-length 99` to `pre-commit-config.yaml` would have worked as well, but with your solution this should also work from cli if you want to check flake8 manually).